libnavigation-core/src/main/java/com/mapbox/navigation/core/replay/route/ReplayRouteLocationEngine.kt

```diff
@@ -19,20 +19,19 @@ import java.util.ArrayList
 /**
  * Mock [LocationEngine] for replaying route movement through [DirectionsRoute]
  *
- * @param logger [Logger](optional)
- * @param converter ReplayLocationConverter
+ * @param logger [Logger] (optional)
  */
 class ReplayRouteLocationEngine @JvmOverloads constructor(
-    val logger: Logger? = null,
-    private val converter: ReplayLocationConverter = ReplayRouteLocationConverter(
-        DEFAULT_SPEED,
-        DEFAULT_DELAY
-    )
+    val logger: Logger? = null
 ) : LocationEngine, Runnable {
 
     private var speed = DEFAULT_SPEED
     private var delay = DEFAULT_DELAY
     private val handler: Handler = Handler()
+    private val converter: ReplayLocationConverter = ReplayRouteLocationConverter(
```

Comment:
Could we make [`ReplayLocationConverter`](https://github.com/mapbox/mapbox-navigation-android/blob/master/libnavigation-core/src/main/java/com/mapbox/navigation/core/internal/replay/route/ReplayLocationConverter.kt) `internal` and move it out from the `internal` package (`internal.route.replay`) into `core.replay.route`?

Reply:
oh yeah, removed it from the internal package